src/sass/partials/components/_forms.scss

```diff
@@ -181,18 +181,12 @@
 .required {
 
   &--save {
-
-    .mark-save {
-      background-color: setting-get('primary support color');
-      font-weight: setting-get('normal font weight');
-    }
+    background-color: setting-get('primary support color');
```

Comment:
Are there any checks in place to ensure that the text color is accessible on the background color?